src/ovp_pipeline/autopilot/daemon.py

```
@@ -482,14 +482,38 @@ def _run_refine(self):
         subprocess.run(breakdown_cmd, capture_output=True, cwd=str(self.vault_dir))
 
     def _run_knowledge_index_refresh(self):
-        """刷新派生 knowledge.db。"""
+        """刷新派生 knowledge.db，并落地 Phase 38 的 Crystal + Working Memory。
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)